source/includes/_gettingstarted.md

```diff
@@ -11,7 +11,13 @@ skynet upload "./image.jpg"
 ```
 
 ```javascript--browser
-// TODO
+import { upload } from "skynet-js";
+
+try {
+  const { skylink } = await upload("https://siasky.net", "./image.jpg");
+} catch (error) {
+  console.log(error)
+}
```

Comment:
This will not work, check the example in the skynet-js repo

Reply:
Why not? What should I do instead?